rpc/types_executables.go

```diff
@@ -0,0 +1,362 @@
+package rpc
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+type CasmCompiledContractClass struct {
+	EntryPointsByType CasmEntryPointsByType `json:"entry_points_by_type"`
+	ByteCode          []*felt.Felt          `json:"bytecode"`
+	Prime             []*felt.Felt          `json:"prime"`
+	CompilerVersion   string                `json:"compiler_version"`
+	Hints             Hints                 `json:"hints"`
```

Comment:
I did that because `hints` is a [tuple](https://github.com/starkware-libs/starknet-specs/blob/19ab9d4df4ae3acc1a52cde5a43a7cace08bcc4b/api/starknet_executables.json#L92) of [intenger, hint] and Golang doesn't allow an array of multiple types. So I created a struct having these two fields and two helper methods for getting the values: the `Value()` that returns both values the way Golang usually does (int, []Hint) and the `Tuple()` that returns an `[2]any` array in case someone needs it in this format